### PR TITLE
Delete sandbox placeholder message

### PR DESCRIPTION
**Pull request by adamawang**
## Description and Context

Add a placeholder command for `hs sandbox delete` where we display a message telling the user to go to their HubSpot portal to delete the account. 

## Screenshots

Screen Shot 2022-05-17 at 6 39 48 PM

## TODO

We will eventually replace this with an actual delete command once `LocalDevAuth` is updated to include hubTypes. Issue tracked here: https://git.hubteam.com/HubSpot/hubspot-cli-issues/issues/327

## Who to Notify

@brandenrodgers @erincouse

### CODE REVIEW

File: packages/cli-lib/lang/en.lyaml [hunk 1]
```diff
@@ -578,6 +578,18 @@ en:
                 instructions: "To update CLI permissions for \"{{ accountName }}\":
                   \n- Go to {{ url }}, deactivate the existing personal access key, and create a new one that includes Sandboxes permissions.
                   \n- Update the CLI config for this account by running `hs auth` and entering the new key.\n"
+          delete:
+            describe: "Delete a sandbox account"
+            examples:
+              default: "Deletes the sandbox account named MySandboxAccount."
+            success:
```
Comment (brandenrodgers) [inline packages/cli-lib/lang/en.lyaml:585]: Looks like the success and error keys aren't being used. Is that intentional because they'll be used later?

**Review by brandenrodgers — APPROVED** [review]
One thing that could potentially make this a little more useful to the user would be to link into hubspot, but this update is still a step forward so I'm okay with merging 👍

**Comment by adamawang** [comment]
Without the hierarchy thing, I'm not able to generate an accurate link to the sandboxes page since the URL requires that parent hub ID... I might as well just add a link to `app.hubspot.com` to at least make it clear on what hubspot website to go to

File: packages/cli-lib/lang/en.lyaml (hunk 1 above)
Comment (adamawang) [reply packages/cli-lib/lang/en.lyaml:585]: Yeah, I had a bunch of code that I deleted as placeholder but left the translations in for now. Hopefully we'll get around to completing this soon, but totally happy to remove if we don't want dead end strings lying around

Comment (brandenrodgers) [reply packages/cli-lib/lang/en.lyaml:585]: Makes sense. I'll leave the decision up to you. If you're confident that you'll update this soon-ish then I'm okay with keeping it.

Comment (adamawang) [reply packages/cli-lib/lang/en.lyaml:585]: Yep! cms dev infra team just got back to me with some info on how best to start working on the auth part, will start on it next Monday

**Pull request merged**